ecs-logs-collector.sh

```diff
@@ -496,6 +499,58 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker "${info_system}"/docker/docker-sysconfig
```

Comment:
nit: I think most of the other files are prefixed to gather them together, making this `sysconfig-docker` instead.

Reply:
Done!